src/components/basicComponents/ShowSuggestions.vue

```diff
@@ -2,12 +2,24 @@
   <div>
     <IconSuggestions />
     <p>
-      <span>dynamic number here</span>
-      Suggestions
+      <span>{{ feedbacksLength }}</span>
+      <span> Suggestion</span>
```

Comment:
I guess you have here 3 spans for styling purposes. If you don't need all of them remove one or two :)

Reply:
I left only one and put the ternary conditional in interpolation, is that way correct?